docs/gathered-data.md

```diff
@@ -52,17 +52,29 @@ The CRD sizes above are in the raw (uncompressed) state.
 
 ## CephCluster
 
-collects statuses of the`cephclusters.ceph.rook.io` resources
+Collects statuses of the`cephclusters.ceph.rook.io` resources
 from Openshift Data Foundation Stack.
 
-API Reference:
+### API Reference:
+https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
 
-	https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+### Sample data:
+docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
 
-* Location in archive: config/storage/<namespace>/<name>.json
-* Id in config: clusterconfig/ceph_cluster
-* Since versions:
-  - 4.12+
+### Location in archive:
+| Version | Path													 |
+| ------- | ----------------------------------------------------- |
+| <= 4.11 | config/storage/{namespace}/cephclusters/{name}.json 	 |
+| 4.12+   | config/storage/{namespace}/cephclusters/{name}.json 	 |
```

Comment:
well this should be 4.13+ right? And then we will need to update with some 4.12.z version, I guess.

Reply:
Not 100% yet, I was double checking in our channel if we will bp to already push it with the correct version.

Reply:
Yes, but first of all I would start with correct versions in the master branch